### PR TITLE
Delete execution event

### PR DESCRIPTION
**Pull request by Cheshiriks**
Delete execution event

### CODE REVIEW

File: db/v-2/events/event_scheduler.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="event-delete-execution-1" author="frolov" context="prod">
```
Comment (nulls) [inline db/v-2/events/event_scheduler.xml:8]: what's about context `scheduler`?

Comment (Cheshiriks) [reply db/v-2/events/event_scheduler.xml:8]: If the context is non-existent, then the event is not created. But we used to have a Prod context, and after we changed it, the base probably didn’t drop, so the events weren’t deleted.

Comment (nulls) [reply db/v-2/events/event_scheduler.xml:8]: maybe then change `scheduler` to `prod` for existed scripts?

Comment (akuleshov7) [reply db/v-2/events/event_scheduler.xml:8]: how liquibase will work with such change of a context?

**Review by akuleshov7 — APPROVED** [review]

File: db/v-2/events/event_scheduler.xml (hunk 1 above)
Comment (nulls) [reply db/v-2/events/event_scheduler.xml:8]: yeap, DATABASECHANGELOG contains the column `contexts` . So changing context will be a problem

**Review by nulls — APPROVED** [review]

**Pull request merged**